### DISCUSSION TITLE
Simple suggestion: remember sauce.

### DISCUSSION THREAD

**Discussion by White-Raven** [getmango/Mango#205] [Ideas]
Hi, I'm just wondering if it's something that goes against your project's optic, or if it's just that you haven't come around yet to do it, but:

since we can basically tell mango "here's the thing I wanna read comfy, batch-download it", couldn't it be possible to just have mango retain that original link, so whenever feeling like it, we can just click a button for it to batch-check if there are new pages/chapters out, and it updates the whole library from the same sources we launched the batch-downloads with at first?

**Comment by Nerothos** [comment]
That's part of #170 
That's something that need to be done by plugins, not base mango
